api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -180,6 +202,7 @@ type FelixConfigurationSpec struct {
 	// Calico's rules from being bypassed. If you switch to append mode, be sure that the other rules in the chains
 	// signal acceptance by falling through to the Calico rules, otherwise the Calico policy will be bypassed.
 	// [Default: insert]
+	// +kubebuilder:validation:Enum=insert;append
```

Comment:
could it be that we also allow `Insert` or `Append` ? :thinking:

Reply:
your brought up a good point - I was thinking everything is case-insensitive. However looks like it's not. Just checked the config_params.go and looks like the values for this one is reflecting what has been defined in the code:

`ChainInsertMode             string `config:"oneof(insert,append);insert;non-zero,die-on-fail"`